apps/give-base/src/components/carbonchart.tsx

```
@@ -0,0 +1,101 @@
+  p50: { background: 'linear-gradient(to right, #1a56db 0%, #1a56db 50%, transparent 50%, transparent 100%)' },
+  p60: { background: 'linear-gradient(to right, #1a56db 0%, #1a56db 60%, transparent 60%, transparent 100%)' },
+  p70: { background: 'linear-gradient(to right, #1a56db 0%, #1a56db 70%, transparent 70%, transparent 100%)' },
+  p80: { background: 'linear-gradient(to right, #1a56db 0%, #1a56db 80%, transparent 80%, transparent 100%)' },
+  p90: { background: 'linear-gradient(to right, #1a56db 0%, #1a56db 90%, transparent 90%, transparent 100%)' }
+} as const
+
+//const CarbonChart = ({
+function CarbonChart({ title, goal=100, value=0, max100=true }: ChartType) {
+  console.log('CHART', goal, value)
+  const max = (max100 ? 100 : goal)
+  const pct:any = {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)